### ISSUE TITLE
Don’t know how to ingest data to use the model

### ISSUE THREAD

**Issue by vionwinnie** [lucidrains/mlm-pytorch#4]
In the markdown, you mentioned `loss=trainer(data)` . As a new user, I’m still figuring out how to use the models with custom data. Maybe I can add a toy_problem.py as examples and create a PR? Would really appreciate your help with this.

**Comment by neel04** [comment]
yeah @lucidrains can you provide some insight as to how should we feed the data? I assume I have to construct a data generator to load each line of my dataset, convert it to integers and feed it as-it-is. am I correct?
